website/sidebars.js

```diff
@@ -14,7 +14,8 @@
     'Getting-Started',
     'Installation',
     {'Use Cases': [{type: 'autogenerated', dirName: 'Use-Cases'}]},
-    {'Examples': [{type: 'autogenerated', dirName: 'Examples'}]},
+    // {'Examples': [{type: 'autogenerated', dirName: 'Examples'}]},
+    'Examples',
```

Comment:
We don't have "Examples" any more at this place. @qingyun-wu could you confirm?

Reply:
Yes, we could remove the sidebar `Examples`,  it fails to compile because the directory examples doesn't exist

Reply:
@sonichi Yes. We need to remove it. We don't have "Examples" any more.

Reply:
Removed